CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v31.0.2...master)
 
+* General:
+    * The `regex` crate is no longer required, making the Glean release binary
+      about 0.5MB smaller.
```

Comment:
I would be a bit more careful of the phrasing here. We deliver to so many different platforms that a blank size statement might be misleading.

Reply:
Good point.  I changed this to just "smaller"